.github/PULL_REQUEST_TEMPLATE/python.md

```diff
@@ -0,0 +1,169 @@
+**Changes:**
+
+* IoT3 SDK:
+    * core: ...
+    * mobility: ...
+* applications:
+    * iqm: ...
+    * ...
+
+Closes: #
+
+---
+**How to test:**
+
+_**Note:** in the following, lines starting with `$` are to be executed on your machine, as a non-root user; lines starting with `(docker)$` are to be executed in the Docker container; lines starting with `(docker)🐍 $` are to be executed in the Docker container, in the python venv._
```

Comment:
you've got a duck smiley in your text :D.

Reply:
It is not a duck, it is a snake (or a python if you squint a bit).